FWCore/Utilities/src/MallocOpts.cc

```diff
@@ -162,6 +162,7 @@ namespace edm
     error_message_.clear();
     changed_ = false;
 
+#ifndef __SANITIZE_ADDRESS__
```

Comment:
Is address sanitizing a function of the operating system or of how the operating system is configured? Basically I'm asking if this is really a compile time switch or should really be a run time one.